TenantFile/Api/Controllers/UserController.cs

```diff
@@ -21,18 +23,54 @@ public async Task MakeUserAdmin([FromQuery] string userUid)
         }
 
         [HttpGet("/users/list")]
-        public async Task ListUsers()
+        public async Task<UserListResult> ListUsers()
```

Comment:
Not necessarily specific to this line but how do you feel about using the `[ApiController]`and `[Route("api/[controller]")]` attributes on the class and letting the name of the method be the route with `[HttpGet("[action]")] `  So this would become {baseUrl}/User/ListUsers or whatever we decide to call it.